src/main/java/seedu/address/model/recipe/RecipeContainsKeywordPredicate.java

```diff
@@ -12,7 +13,11 @@
     private final List<String> keywords;
 
     public RecipeContainsKeywordPredicate(List<String> keywords) {
-        this.keywords = keywords;
+        this.keywords = List.of(keywords.toArray(new String[]{}));
```

Comment:
What's the purpose of this line actually? Does doing this create a "hard" copy of the values in the keywords parameter?

Reply:
@aidanaden This change is more of a defensive change to make keywords an immutable list so that its values can't be changed if accessed through `RecipeContainsKeywordPredicate`.